File: bubble_sort.py
def bubble_sort(arr):
    for i in range(len(arr)):
        for j in range(len(arr) - 1 - i):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]


x = [5,2,8,4,3]
bubble_sort(x)
print(x)

File: bubble_sort.cpp
#include <iostream>

using namespace std;

void bubble_sort(int arr[], int size){
    for (int i = 0; i < size; i++)
        for (int j = 0; j < size - 1 - i; j++){
            if(arr[j] > arr[j+1]){
                int temp = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = temp;
            }
        }
}

int main(){
    int x[] = {6,3,8,22,5,4,99,21};
    int size = sizeof(x)/sizeof(int);
    bubble_sort(x, size);

    for (int i = 0; i < size; i++)
        cout<<x[i]<<endl;
    return 0;
}